Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/BoundingBox/BoundingBox.cs

```diff
@@ -2,6 +2,7 @@
 // Licensed under the MIT License. See LICENSE in the project root for license information.
 
 using Microsoft.MixedReality.Toolkit.Input;
+using Microsoft.MixedReality.Toolkit.Utilities;
```

Comment:
Documentation page?

Reply:
I think @thalbern is in the process of refactoring BoundingBox. So might make sense to add then?

Reply:
I mean for the ScaleMeshEffect

Reply:
Ah gotcha, yeah I can add a blub to the MRTK/Shader page.